posts/hybrid-reference-profiles/index.qmd

```diff
@@ -0,0 +1,70 @@
+---
+title: "Creating hybrid reference profiles for InSituType"
+author:
+  - name: Patrick Danaher
+    orcid: 0000-0002-2844-5883
+    affiliations:
+      - ref: nstg
+      - ref: patrickjdanaher
+date: "2024-08-21"
+date-modified: "2024-08-21"
+categories: [cell typing]
+draft: false
+image: figures/heatmap_tile.png
+---
```

Comment:
(Optional) If you add a 'description' line, this will show up on the blog view, making it slightly cleaner than showing however much of the first paragraph fits. Something like this between lines 13 and 14:
`description: "A short post on reference profiles."`